src/lolex-src.js

```diff
@@ -193,6 +191,24 @@ function withGlobal(_global) {
             }
         }
 
+        function ClockDate() {
+            var args = Array.prototype.slice.call(arguments);
+
+            if (args.length === 0) {
```

Comment:
I don't understand why this case needed to be moved out? Is there something particular about this case that would cause different behavior when left in the switch case?

Reply:
Thanks. I couldn't remember. You are right it shouldn't be a separate case.